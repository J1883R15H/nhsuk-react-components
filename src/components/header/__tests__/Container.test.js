import React from 'react';
import { shallow } from 'enzyme';
import Container from '../components/container';

describe('Header.Container', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Container>Child</Container>);
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
