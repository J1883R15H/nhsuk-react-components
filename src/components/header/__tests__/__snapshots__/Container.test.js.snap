// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.Container matches snapshot 1`] = `
<div
  className="nhsuk-width-container nhsuk-header__container"
>
  Child
</div>
`;
